content/docs/command-reference/plots/show.md

```diff
@@ -1,58 +1,65 @@
 # plots show
 
-Generate a plot image from from a [plot metrics](/doc/command-reference/plots)
-file.
+Generate [plot](/doc/command-reference/plots) from a metrics file.
 
 ## Synopsis
 
 ```usage
-usage: dvc plots show [-h] [-q | -v] [-t [TEMPLATE]] [-f FILE]
-                     [-s SELECT] [-x X] [-y Y] [--stdout]
-                     [--no-csv-header] [--no-html] [--title TITLE]
-                     [--xlab XLAB] [--ylab YLAB] [datafile]
+usage: dvc plots show [-h] [-q | -v] [-t <path>] [-o <path>]
+                      [-x <field>] [-y <field>] [--no-csv-header]
+                      [--show-vega] [--title <text>]
+                      [--x-label <text>] [--y-label <text>]
+                      targets [targets ...]
 
 positional arguments:
-  datafile              Metrics file to visualize
+  targets               Metrics files to visualize.
 ```
 
 ## Description
 
 This command provides a quick way to visualize metrics such as loss functions,
-AUC curves, confusion matrices, etc. Please see `dvc plots` for information on
-the supported data formats and other relevant details about DVC plots.
+AUC curves, confusion matrices, etc. One of more `targets` are required as
+argument to this command.
+
+This command can work with target metric files that are versioned directly with
```

Comment:
I think it's not important to clarify it there - since it reads any file literally that you pass it.

`Target - path to a file with metrics`